C/common/asset_tracking.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+ * FogLAMP asset tracking related
+ *
+ * Copyright (c) 2019 Dianomic, LLC
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Amandeep Singh Arora
+ */
+
+#include <logger.h>
+#include <asset_tracking.h>
+
+using namespace std;
+
+
+AssetTracker *AssetTracker::instance = 0;
+
+AssetTracker *AssetTracker::getAssetTracker()
```

Comment:
We should have doxygen style comments on every method

Reply:
Done.